lisa/sut_orchestrator/azure/common.py

```diff
@@ -244,6 +251,22 @@ class VhdSchema(AzureImageSchema):
     vmgs_path: Optional[str] = None
 
 
+@dataclass_json()
+@dataclass
+class CommunityGalleryImageSchema(AzureImageSchema):
+    image_gallery: str = ""
+    image_definition: str = ""
+    image_version: str = ""
+    location: str = ""
+
+    def __hash__(self) -> int:
+        return hash(
```

Comment:
hash is to generate an unique string, so it doesn't need fixed strings. Split chars are enough, like 

```python
            f"{self.image_gallery}/{self.image_definition}/{self.image_version}"
```